app/views/checkout/_details.html.haml

```diff
@@ -93,7 +93,7 @@
             %em.fees= payment_or_shipping_price(shipping_method, @order)
             - display_ship_address = display_ship_address || (ship_method_is_selected && shipping_method.require_ship_address)
         %div.checkout-input{"data-shippingmethod-target": "shippingMethodDescription", "data-shippingmethodid": shipping_method.id , style: "display: #{ship_method_is_selected ? 'block' : 'none'}" }
-          #distributor_address.panel
+          -# #distributor_address.panel
             - if shipping_method.description.present?
```

Comment:
We prefer not to keep commented-out code, so you can go ahead and delete the code as per the requirements.  If it's required in the future, it shouldn't be hard put back (we can check the git history). Because the codebase is so big, it  would be too hard to manage if we kept unused code.

However, from the picture it looks like we want to keep the **shipping method description**, and only remove the **"ready for"** section. I think that means removing lines 99-103.

Reply:
Will change the code in order to remove only the "ready for" section as noted by dacook.

Reply:
Your assumption is correct, i have to delete lines 99-103.